edk2toollib/uefi/edk2/parsers/base_parser.py

```diff
@@ -240,12 +240,17 @@ def ComputeResult(self, value, cond, value2):
             return (ivalue != ivalue2) and (value != value2)
 
         # check to make sure we only have digits from here on out
```

Comment:
It seems this line should go after this block and before the `isdigit()` checks below?

Reply:
All three of the checks below this line are to insure we only have digits from here on out (verifying value is not a string representation of true / false, and isinstance / isdigit checks).

I can't rely soley on isinstance and isdigit because `self.ConvertToInt` converts the any TRUE / FALSE strings to 1 / 0, so we have to make sure the origin value was not a TRUE/FALSE also.